rmf_demo_tasks/rmf_demo_tasks/dispatch_clean.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import uuid
+import time
+import argparse
+
+import rclpy
+from rclpy.node import Node
+from rmf_task_msgs.srv import SubmitTask
+from rmf_task_msgs.msg import TaskType, Clean
+
+###############################################################################
+
+
+class TaskRequester:
+
+    def __init__(self, argv=sys.argv):
+        parser = argparse.ArgumentParser()
+        parser.add_argument('-cs', '--clean_start', required=True,
+                            help='clean start waypoint')
+        parser.add_argument('-st', '--start_time',
+                            help='Start time from now in secs, default: Now',
+                            type=int, default=0)
+
+        self.args = parser.parse_args(argv[1:])
+        self.node = rclpy.create_node('task_requester')
+        self.submit_task_srv = self.node.create_client(
+            SubmitTask, '/submit_task')
+
+    def generate_task_req_msg(self):
+        req_msg = SubmitTask.Request()
+        req_msg.description.task_type.type = TaskType.TYPE_CLEAN
+
+        clean = Clean()
+        clean.start_waypoint = self.args.clean_start
+        req_msg.description.clean = clean
+
+        ros_start_time = self.node.get_clock().now().to_msg()
```

Comment:
Refer to comment on `use_sim_time`

Reply:
Thanks for raising this issue that I overlooked it. I have added an optional flag `--disable_sim_time` to those scripts, so the default will to use sim time. Tested and works.